### PR TITLE
Add success and score to statements

### PR DESCRIPTION
**Pull request by garemoko**
todo:

* consider changing verb based on success
* docs
* tests

### CODE REVIEW

**Comment by garemoko** [comment]
Not sure why this is conflicting. Will look once #19 merges.

**Comment by garemoko** [comment]
The verb display looks sufficiently complicated that I'm going to leave the verb alone for now. Moving on to docs and tests.

**Comment by ryansmith94** [comment]
What's wrong with the verb display?

**Comment by garemoko** [comment]
@ryansmith94 There's some clever stuff in preparation for localization that makes it more complex to change the verb for an event based on data.

**Comment by ryansmith94** [comment]
Why does the verb display need to be changed based on data?

**Comment by garemoko** [comment]
@ryansmith94 it doesn't *need* to be, but it might be nice to use "passed", "failed" and "completed" based on the value of success (true, false or null).

**Comment by garemoko** [comment]
@ryansmith94 this one is also good for review now

**Comment by ryansmith94** [comment]
Ok I think, we should leave the variable verbs for now.

File: tests/AttemptCompletedTest.php [hunk 1]
```diff
@@ -35,7 +39,10 @@ protected function assertOutput($input, $output) {
         $this->assertAttempt($input, $output['context']['contextActivities']['grouping'][2]);
         $this->assertObject('course', $input, $output['context']['contextActivities']['grouping'][1]);
         $this->assertObject('module', $input, $output['object']);
-        $this->assertEquals($input['attempt_result'], $output['result']['score']['raw']);
+        $this->assertEquals($input['attempt_score_raw'], $output['result']['score']['raw']);
+        $this->assertEquals($input['attempt_score_min'], $output['result']['score']['min']);
+        $this->assertEquals($input['attempt_score_max'], $output['result']['score']['max']);
+        $this->assertEquals($input['attempt_score_scaled'], $output['result']['score']['scaled']);
```
Comment (ryansmith94) [inline tests/AttemptCompletedTest.php]: I think there needs to be an additional assertion here for `attempt_success`.

**Comment by garemoko** [comment]
@ryansmith94 done

**Comment by ryansmith94** [comment]
Lovely thanks :)

**Pull request merged**